handler.go

```diff
@@ -162,11 +162,45 @@ func withHTTPContext(ctx context.Context) context.Context {
 	return WithContext(ctx, hctx)
 }
 
+func setTracingContext(config map[string]string, header http.Header) tracingCtx {
+	if config["OCI_TRACING_ENABLED"] == "0" {
+		// When tracing is not enabled then we
+		// assign empty tracing context to
+		// the context
+		return tracingCtx{}
+	}
+	tctx := tracingCtx{
+		traceCollectorURL: config["OCI_TRACE_COLLECTOR_URL"],
+		traceId:           header.Get("x-b3-traceid"),
+		spanId:            header.Get("x-b3-spanid"),
+		parentSpanId:      header.Get("x-b3-parentspanid"),
+		flags:             header.Get("x-b3-flags"),
+		appName:           config["FN_APP_NAME"],
+		fnName:            config["FN_FN_NAME"],
+	}
+
+	isSampled, err := strconv.ParseBool(header.Get("x-b3-sampled"))
```

Comment:
if that header is not there we will get back an error from the ParseBool calll that means we set up the tctx.sampled = true. 
I think that is correct, can you just confirm that that is the logic we want?